pkg/tempest/job.go

```diff
@@ -41,6 +43,9 @@ func Job(
 						RunAsUser:  &runAsUser,
 						RunAsGroup: &runAsGroup,
 						FSGroup:    &runAsGroup,
+						SELinuxOptions: &corev1.SELinuxOptions{
```

Comment:
@afazekas, @jpodivin  do you think that this something I should be worried about?

Reply:
First I wouldn't hardcode the level. Second, I'm not sure if least sensitive is the right setting. But given it's just tests it may be. 
Still, I wouldn't use the default level from documentation. 

More important is the fact tha you are not setting `MustRunAs` in the selinuxcontext, which could lead to your label being ignored.

Reply:
As we discussed, I introduced a new SELinuxLevel option that can by used by the user to overwrite the default `SELinuxLevel` which is set to `s0:c478,c978` by the test-operator.